devolutions-gateway/src/api/mod.rs

```diff
@@ -21,7 +21,7 @@ pub fn make_router<S>(state: crate::DgwState) -> axum::Router<S> {
         .nest("/jet/jrec", jrec::make_router(state.clone()))
         .nest("/jet/config", config::make_router(state.clone()))
         .nest("/jet/session", session::make_router(state.clone()))
-        .nest("/jet/sessions", sessions::make_router(state.clone()))
+        .nest("/jet/sessions", session::make_router(state.clone()))
```

Comment:
issue: Any reason for this? This looks very problematic (creating the wrong router).

Reply:
this is a mistake, thank you for point it out!

Reply:
updated